config/template_config.toml

```
@@ -11,9 +12,9 @@
 # he - Hebrew, by https://github.com/netanelkoli
 # es_mx - Spanish (Mexican), by https://github.com/mastersuv
 # The lanugages that messages can be displayed in
-enabled_languages = ["it", "en", "uk", "ru", "zh_cn", "he", "es_mx"]
+enabled_languages = ["br","it", "en", "uk", "ru", "zh_cn", "he", "es_mx"]
```

Same thing as earlier.

```suggestion
enabled_languages = ["it", "en", "uk", "ru", "zh_cn", "he", "es_mx", "br"]
```

Additionally, shouldn't Brasilian be `pt_br`?